pages/03.windows/02.installation/docs.en.md

```diff
@@ -48,6 +48,8 @@ To use Adguard on a PC or tablet under Microsoft Windows, you should download th
 
```

Comment:
Something is not okay with the formatting:
https://monosnap.com/file/NXAVgrMLdyJTES7il3nrUqgh0cjBlb